src/rotten-tomatoes/SpoilTomatoes.js

```diff
@@ -2,41 +2,40 @@ class SpoilTomatoes {
 
     constructor(grid) {
         this.grid = grid;
-        this.colLength = 0;
-        this.rowLength = 0;
         this.getAdjacents = this.getAdjacents.bind(this)
+        this.getRottenTomatoes = this.getRottenTomatoes.bind(this)
```

Comment:
Didn't catch this before. Why do you need to do this? Its unnecessary in this context.

Reply:
A habit i picked up from using class in React